packages/dds/tree/src/shared-tree-core/transaction.ts

```diff
@@ -138,7 +155,13 @@ export class TransactionStack implements Transactor, IDisposable {
 
 	public start(): void {
 		this.ensureNotDisposed();
-		this.#stack.push(this.#onPush?.());
+		const onPushCurrent = hasSome(this.#stack)
+			? // No `at` method available
+				// eslint-disable-next-line unicorn/prefer-at
+				this.#stack[this.#stack.length - 1]?.onPush
```

Comment:
We have a `getLast` function which is designed to work alongside `hasSome` - you can use that here.